source/implementations/f7/Meadow.F7/Interop/Interop.cell.cs

```diff
@@ -1,16 +1,111 @@
 ﻿using System.Runtime.InteropServices;
+using System.Text.RegularExpressions;
+using System.Collections.Generic;
+using System.Text;
 using System;
 
+using Meadow.Networking;
 namespace Meadow.Core;
 
 internal static partial class Interop
 {
+    internal  enum CellNetworkMode { GSM =0,CAT_M1_BG77 = 7,CAT_M1 = 8, NB_IoT = 9  };
```

Comment:
There's an extra space before `enum CellNetworkMode`, please adjust it.

Reply:
`GSM =0,CAT_M1_BG77 = 7,CAT_M1 = 8, NB_IoT = 9`
There's an inconsistency here, there are some spaces after a comma, but sometimes there isn't.